Dockerfile

```
@@ -0,0 +1,4 @@
+FROM golang:1.18.4-buster
+ENTRYPOINT ["/tagger", "--config", "/tmp/tagger.yml"]
+COPY tagger /tagger
```

Adding onto my other comment:
```suggestion
COPY tagger /usr/bin/tagger
ENTRYPOINT ["/usr/bin/tagger"]
```